src/m365/aad/commands/m365group/m365group-user-list.ts

```diff
@@ -73,17 +103,36 @@ class AadM365GroupUserListCommand extends GraphCommand {
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     try {
-      const isUnifiedGroup = await aadGroup.isUnifiedGroup(args.options.groupId);
+      if (args.options.role === 'Guest') {
+        this.warn(logger, `Value 'Guest' for the option role is deprecated.`);
```

Comment:
When showing a deprecation, it would be helpful to show users what they should do instead, to help them change their scripts without having to search for answers manually.